rnd/autogpt_server/autogpt_server/executor/manager.py

```diff
@@ -58,6 +58,10 @@ def get_log_metadata(
     }
 
 
+def get_log_prefix(graph_eid: str, node_eid: str, block_name: str = "-"):
+    return f"[ExecutionManager][graph-eid-{graph_eid}|node-eid-{node_eid}|{block_name}]"
```

Comment:
log metadata has graph-id and node-id too, can we just generate the log prefix from there?

Reply:
I just put back in the same as what we had here previously - we can add more in, in a follow up PR for sure